com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -202,6 +202,12 @@ public bool IsCameraProjectionMatrixFlipped()
         public int mainLightShadowmapHeight;
         public int mainLightShadowCascadesCount;
         public Vector3 mainLightShadowCascadesSplit;
+        /// <summary>
+        /// Main light last cascade shadow fade border.
+        /// Value represents the width of shadow fade that ranges from 0 to 1.
+        /// Where value 0 is used for no shadow fade.
+        /// </summary>
+        public float mainLightShadowCascadeBorder;
```

Comment:
Yes and yes. My though behind this was - it is safer to deprecate it later instead of introducing generic property which has empty values. Should I change it to vector4, not sure about the naming :/?